Fixed the typo at the example code of README.md.

kelp404 opened the pull request:
Hello,

Thank you for this open source project.
I fixed the typo at the example code of README.md.

givanse commented:
CI is failing because there is a linter error:

> 1:8  error  'RouteRecognizer' is defined but never used  no-unused-vars

its fixed in either one of these: https://github.com/pretenderjs/pretender/pull/256 https://github.com/pretenderjs/pretender/pull/257

kelp404 commented:
I just modified on line at README.md.
I changed the head at my repository. Then re-commit it.
But the Travis CI test the merged code. Your repository is already failure.
<img width="1042" alt="螢幕快照 2019-04-29 上午6 54 12" src="https://user-images.githubusercontent.com/1680868/56871109-c57e7000-6a4b-11e9-8de2-eaed191c073b.png">

xg-wang commented:
Fixed the lint CI failure and merged, thank you @kelp404
